frontend/src/metabase/visualizations/components/settings/ChartSettingIconRadio.styled.tsx

```diff
@@ -1,16 +1,6 @@
 import styled from "@emotion/styled";
-import Icon from "metabase/components/Icon";
+import Button from "metabase/core/components/Button";
 
-import { color } from "metabase/lib/colors";
-
-interface RadioIconProps {
-  isSelected: boolean;
-}
-
-export const RadioIcon = styled(Icon)<RadioIconProps>`
-  margin-left: 1rem;
-  cursor: pointer;
-  user-select: none;
-
-  ${props => props.isSelected && `color: ${color("brand")}`}
+export const IconButton = styled(Button)`
```

Comment:
I think you should rename the file too, otherwise, it's going to be a bit hard to search for the component since we all assume both the file name and the component name should be the same.